models/stdp_pl_connection_hom.cpp

```diff
@@ -65,6 +66,7 @@ STDPPLHomCommonProperties::set_status( const DictionaryDatum& d,
   CommonSynapseProperties::set_status( d, cm );
 
   updateValue< double >( d, "tau_plus", tau_plus_ );
+  tau_plus_inv_ = 1. / tau_plus_;
```

Comment:
Something to remember for further reviews and checks, since apparently that's slipped in, in many places earlier and only became apparent now.